SEP-0002.md

```diff
@@ -1,9 +1,14 @@
-* author: Steven Christe
-* contact email:
-* date-creation: 2014/02/16
-* date-last-revision: 2014/06/09
-* type: process
-* status: accepted
+# SEP-0002 -- SunPy Organization Definition
+
+| SEP           | 2 |
+|---------------|---|
+| title         | SunPy Organization Definition |
+| author(s)     | Steven Christe |
+| contact email | steven.d.christe@nasa.gov |
+| date-creation | 2014/02/16 |
+| type          | process |
+| discussion    | None |
```

Comment:
Is "None" the right way to describe this?  There was discussion, just no link.  Note that in SEP-0004, this spot is simply empty instead of saying anything.